README.md

```diff
@@ -68,6 +69,24 @@ This software package includes solvers for continuous optimization problems. The
     <img src="ccvm_simulators/ccvmplotlib/images/success_prob_plot_example.png" width="250">
 </p>
 
+
+## Architecture and Workflow Overview
+#### Example Workflow
+The workflow diagram showcases, step-by-step, how the CCVM simulators can be
+used to solve a problem.
+
+<p align="center">
+    <img src="diagrams/example_workflow.png">
+</p>
+
+#### High-Level Class Diagram
+This diagram offers a concise visualization of the main classes and their interactions within the `ccvm_simulators` package.
+
+<p align="center">
+    <img src="diagrams/high_level_class_diagram.png">
+</p>
+
+
```

Comment:
I know my previous comment said that we would want the extra diagrams in their respective READMEs but since those READMEs don't exist, can you embed those images here with a little explanation that the boxes in the diagram above (high_level_class_diagram) extend like so? I want to make sure they get seen!

Reply:
Sure. I've added a small description and the graphs in the README.